### ISSUE TITLE
Render OAS schemas

### ISSUE THREAD

**Issue by avaidyam** [cloud-annotations/docusaurus-plugin-openapi#16]
Right now, in addition to the OAS info not being rendered as an initial page (see #8), the OAS schemas section is not rendered ass well. It would probably be a single sidebar category called "Schemas" with each model underneath it (and the demo panel would be disabled for these pages).

**Comment by bourdakos1** [comment]
In the link provided, the `schema` seems to be part of the normal operation page as part of the `request body` table. I believe this is the default behavior for `docusaurus-plugin-openapi`?

Am I missing something?

**Comment by avaidyam** [comment]
This feature request is actually to allow non-operations to appear in the sidebar. Specifically, a new section (like `Introduction`, but at the bottom) called `Schemas` whose sub-items are ALL the schemas ONLY, whether used in any operation or not. These schema pages wouldn't have an execute panel since they just display data type + example object info.

**Comment by bourdakos1** [comment]
Oh okay, is this like the "Models" section of Swagger UI: https://petstore.swagger.io?

Couple UX questions:
- Will this just render schemas listed under `components.schemas` or should it also collect schemas that are defined inline from all of the requests?
- Should all of the schemas be part of one `Schemas` page or should it be a `Schemas` category, with a page for each schema?

**Comment by avaidyam** [comment]
Yep! That's the same idea. I think it should only render `components.schemas` and it would be a Schemas category with one page per schema.

**Comment by tikre2910** [comment]
Hi! 
I have the same issue that components.schemas is not rendered in docusaurus (I am also using redocusaurus in addition) and I am just wondering if there is any progress?
Thanks in advance for any replies :)

**Comment by bourdakos1** [comment]
No progress has been made, but happy to accept a pull request :)
